encryption/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.28)
+project(encryption)
+
+set(CMAKE_CXX_STANDARD 14)
+
+add_executable(encryption main.cpp caesar.cpp)
```

Comment:
We are not discuss about it, but I suggest to add all project files to make, not only implementation related